src/datachain/remote/studio.py

```diff
@@ -231,6 +235,40 @@ def _unpacker_hook(code, data):
 
         return msgpack.ExtType(code, data)
 
+    async def tail_job_logs(self, job_id: str) -> AsyncIterator[dict]:
+        """
+        Follow job logs via websocket connection.
+
+        Args:
+            job_id: ID of the job to follow logs for
+
+        Yields:
+            Dict containing either job status updates or log messages
+        """
+        parsed_url = urlparse(self.url)
+        ws_url = urlunparse(
+            parsed_url._replace(scheme="wss" if parsed_url.scheme == "https" else "ws")
+        )
+        ws_url = f"{ws_url}/logs/follow/?job_id={job_id}&team_name={self.team}"
```

Comment:
Why do we need to pass the team name?

Reply:
It seems to be an existing approach for Studio endpoints that takes the team name to check for permission before any other tasks. I reused the same decorator and approach to make sure the permission is correct

Reply:
This doesn't feel right to me. 🤔 In theory, the job identifier and the access token should be the only values needed to check for permission; anything else is a potential footgun in the best case, or an actual footgun in the worst case.

Reply:
Resolving as off-topic, but pinging @dreadatour just to make noise.